### PR TITLE
BUG: CLI is using tests/e2e/pkg

### PR DESCRIPTION
**Pull request by viswanathtadi**
# Description

CLI is using tests/e2e/pkg
Removed the dependency tests/e2e/pkg for CLI
CLI now creates clientsets using helpers in cli/pkg/helpers
Made sure that all unit tests in CLI are passing.

# Checklist 

As the author of this PR, I have:

- Checked that I included or updated copyright and license notices in all files that I altered

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tools/cli/vz/cmd/app/helidon/list.go [hunk 1]
```diff
@@ -52,7 +51,10 @@ func NewCmdAppHelidonList(streams genericclioptions.IOStreams) *cobra.Command {
 
 func listHelidonApplications(cmd *cobra.Command, args []string) error {
 	// connect to the server
-	config := pkg.GetKubeConfig()
+	config, err := helpers.GetKubeConfig()
+	if err != nil {
```
Comment (lake-of-dreams) [inline tools/cli/vz/cmd/app/helidon/list.go:55]: return err, not nil

**Review by lake-of-dreams — DISMISSED** [review]

**Review by bobdonat — DISMISSED** [review]

**Review by markxnelson — DISMISSED** [review]

File: tools/cli/vz/pkg/helpers/kubernetes.go [hunk 2]
```diff
@@ -4,17 +4,78 @@
...
+	if err != nil {
+		return clientset, err
+	}
+	clientset, err = kubernetes.NewForConfig(config)
+	return clientset, err
+}
+
+// Returns kubeconfig from KUBECONFIG env var if set
+// Else from default location ~/.kube/config
+func GetKubeConfig() (*restclient.Config, error) {
```
Comment (markxnelson) [inline tools/cli/vz/pkg/helpers/kubernetes.go:54]: surely we have existing code that does this already?

**Review by markxnelson — COMMENTED** [review]
seems to be code duplication in here

File: THIRD_PARTY_LICENSES.txt [hunk 3]
```diff
@@ -1276,9 +1330,12 @@ github.com/spf13/cobra
 -------- Copyrights
 Copyright © 2020 Steve Francia <spf@spf13.com>
 Copyright © 2015 Steve Francia <spf@spf13.com>.
+Copyright:    copyrightLine(),
```
Comment (markxnelson) [inline THIRD_PARTY_LICENSES.txt:1333]: you need to remove these lines that just have code in them

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: THIRD_PARTY_LICENSES.txt (hunk 3 above)
Comment (lake-of-dreams) [reply THIRD_PARTY_LICENSES.txt]: fixed

**Comment by lake-of-dreams** [comment]
>seems to be code duplication in here
@markxnelson common code moved to pkg repo in https://github.com/verrazzano/pkg/pull/2, please if you can re-review

File: tools/cli/vz/pkg/helpers/environment.go [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+package helpers
+
+import (
+	"context"
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func GetHostnameFromGateway(namespace string, appConfigName string) (string, error) {
```
[the remark this answers is not in the document]
Comment (lake-of-dreams) [reply tools/cli/vz/pkg/helpers/environment.go]: code moved to pkg

File: tools/cli/vz/pkg/helpers/kubernetes.go (hunk 2 above)
Comment (lake-of-dreams) [reply tools/cli/vz/pkg/helpers/kubernetes.go]: code moved to pkg

**Comment by lake-of-dreams** [comment]
one of the mc test is failing, debugging

**Comment by lake-of-dreams** [comment]
all the changes done and tests are passing

**Review by markxnelson — APPROVED** [review]

**Pull request merged**